server/etcdserver/api/etcdhttp/health.go

```diff
@@ -342,6 +373,23 @@ func listToStringSet(list []string) StringSet {
 	return set
 }
 
+func recordMetrics(path, name string, status string) {
+	val := 0
+	if status == HealthStatusSuccess {
+		val = 1
+	}
+	path = strings.TrimPrefix(path, "/")
```

Comment:
```suggestion
```

Reply:
`path` usually starts with `/`, but it is suggested not have the `/` in the counter names.